<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TierWidget</class>
 <widget class="QWidget" name="TierWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Übersicht:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTableView" name="tableTier">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>120</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="tierNeu">
          <property name="text">
           <string>Neues Tier</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tierTot">
          <property name="text">
           <string>Tier löschen</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Informationen über ausgewähltes Tier</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Tier befindet sich im Stall</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboStall"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Tier hat folgende Attribute</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableAttribute">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Tierarzt </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableArzt">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Futtermenge</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableFutterTier">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
